server/build/docker-compose.common.yml

```diff
@@ -87,7 +87,9 @@ services:
       LDAP_DOMAIN: "mm.test.com"
       LDAP_ADMIN_PASSWORD: "mostest"
   elasticsearch:
-    image: "mattermostdevelopment/mattermost-elasticsearch:7.17.10"
+    build:
+      context: .
+      dockerfile: ./Dockerfile.opensearch
```

Comment:
This is also temporary. We'll need a way to swap out these images/containers to run tests on both containers, or have OpenSearch as a separate container defined in this file, though it cannot run at the same time as the elasticsearch container due to the port mapping. We could change that as well though. Open to what others think is the most maintainable route